docs/guides/remove-trusted-relayer.mdx

```diff
@@ -4,26 +4,28 @@
 
 When first deploying with the CLI, new warp routes use a Trusted Relayer ISM out of the box so you don't need to run a relayer or validator.
 
-In order to go to production, you'll need to remove this ISM with the Hyperlane CLI.
+In order to go to production, you'll need to remove this ISM with the [Hyperlane CLI](https://docs.hyperlane.xyz/docs/reference/cli).
 
 ## Prerequisites
 
 - The warp route config generated by `hyperlane warp init`.
-    - This config is used to deploy a warp route. By default, it takes the filepath of `CURRENT_DIR/warp-route-deployment.yaml`.
-- The warp route deployment artifacts generated by `hyperlane warp deploy`.
-    - This deployment artifact file is generated by a warp route deployment. By default, it is written to `$HOME/.hyperlane/deployments/warp_routes/TOKEN/chain1-chain2-config.yaml`.
+    - This config is used to deploy a warp route. By default, it takes the filepath of `CURRENT_DIR/configs/warp-route-deployment.yaml`.
+- The warp route token symbol used in `hyperlane warp deploy`.
+    - The token symbol can be found onchain, or in your local Hyperlane Registry. By default, it is written to `$HOME/.hyperlane/deployments/warp_routes/<tokenSymbol>`.
 - Access to the private key that currently owns the warp route.
 
 :::info
 If you followed the [Deploy a Warp Route](https://docs.hyperlane.xyz/docs/guides/deploy-warp-route) guide, you most likely have deployed a warp route with a trusted relayer set to a signer address.
 :::
 
-To confirm using the Hyperlane CLI, locate your warp route under the `/deployments/warp_routes` directory, and execute the following with `address` set to the `addressOrDenom` of your warp route:
+To confirm using the Hyperlane CLI, locate your token symbol and the chain it is deployed on:
```

Comment:
Pedantic grammar nit: the chain on which it is deployed

Reply:
fixed